lib/user/pages/user_page_success.dart

```diff
@@ -118,410 +104,433 @@ class _UserPageSuccessState extends State<UserPageSuccess> with TickerProviderSt
     final DateTime now = DateTime.now().toUtc();
     final int? currentUserId = context.read<AuthBloc>().state.account?.userId;
 
-    return Center(
-      child: Stack(
-        children: [
-          Column(
-            children: [
-              GestureDetector(
-                onTap: () {
-                  setState(() {
-                    _displaySidebar = !_displaySidebar;
-                  });
-                },
-                onHorizontalDragUpdate: (details) {
-                  if (details.delta.dx < -3) {
+    return BlocProvider<post_bloc.PostBloc>(
+      create: (context) => post_bloc.PostBloc(),
```

Comment:
Does this work? I know we have issues with post actions on the user page because of the different bloc types.

Reply:
Yes it does, We do however need to re-work where and when they are created and used.

Reply:
Yeah, this should work since its essentially creating a new PostBloc instance here rather than fetching an existing PostBloc from the widget tree. I do agree with @ggichure that we need to refactor how they are created and used.